R/client.R

```diff
@@ -5,7 +5,7 @@
 #' @return client handle
 #' @importFrom RProtoBuf P serialize read new
 #' @export
-grpc_client <- function(impl, channel) {
+grpc_client <- function(impl, channel, client_deadline) {
```

Comment:
A few comments on this:

1. change to `function(impl, channel, ..., client_deadline)` - this forces passing it by `name=value` instead of by position.
2. Add default to function parameter - see the other PR on using the options system. Don't want to break existing code if we can avoid it.
3. update docs, and rerun roxygen

Reply:
I committed the updates you asked for and ready for any other suggestions offered by you.

Reply:
I documented the updates and tested them too using the Helloserver and Helloclient example.